src/FriendsPage.tsx

```diff
@@ -12,35 +12,37 @@ const FriendsPage: React.FC = () => {
 
```

Comment:
The only changes in this file are adding the `credential` option to fetch requests. Everything else is prettier's doing